backend/curation/api/v1/curation/collections/common.py

```diff
@@ -150,6 +150,8 @@ def reshape_for_curation_api(
         revision_of=revision_of,
         visibility=get_visibility(collection_version),
     )
+    if reshape_for_version_endpoint:
+        del response["revised_at"], response["revising_in"], response["revision_of"]
```

Comment:
Adding these to the response is more efficient than subtracting from the response.
```python
    if not reshape_for_version_endpoint:
        response.update(revised_at=collection_version.canonical_collection.revised_at,
                        revising_in=revising_in,
                        revision_of=revision_of,)
```